agenda.html

```diff
@@ -0,0 +1,174 @@
+        </div>
+      </div>
+    </nav>
+    <!-- End Navbar Section -->
+  </header>
+  <!-- Start Agenda Section -->
+  <section class="container">
+    <h4 class="text-success text-center"><strong>WDiS POZNAŃ SATELITE EVENT</strong></h4>
+    <h5 class="text-center">13.04.2019, Poznań University od Technology</h5>
+    <h3 class="text-center font-weight-bold">Agenda</h3>
+
+    <table class="table" style="width:100%">
+      <tr>
+        <td class="time">10:00-10:15</td>
+        <td class="font-weight-bold">Opening remarks</td>
+      </tr>
+      <tr>
+        <td class="time">10:15-11:05</td>
+        <td><span class="text-success font-weight-bold">Practical transfer learning for NLP with spaCy and
+            Prodigy</span>
+          <br>INES MONTANI, Founder of Explosion Al</td>
+      </tr>
+      <tr>
+        <td class="time">11:10-12:00</td>
+        <td><span class="text-success font-weight-bold">Deep understanding text-based models</span><br>ANNA WRÓBLEWSKA,
+          PhD<br>Warsaw University of Technology & Wpplica.ai</td>
+      </tr>
+      <tr>
+        <td class="time">12:00-12:30</td>
+        <td class="font-weight-bold">Coffee Break</td>
+      </tr>
+      <tr>
+        <td class="time">12:30-13:20</td>
+        <td><span class="text-success font-weight-bold">How to learn trusthworthy knowledge graphs?<br>Seven problems
+            and seven remedies</span><br>AGNIESZKA ŁAWRYNOWICZ, PhD<br> Poznań University of Technology</td>
+      </tr>
+      <tr>
+        <td class="time">13:25-14:15</td>
+        <td><span class="text-success font-weight-bold">Exploring Speech rhythm and melody<br>through multilayer
+            annotation mining</span><br>KATARZYNA KLESSA,
+          PhD<br>Adam Mickiewicz University in Poznań, Lux-Soft </td>
+      </tr>
+      <tr>
+        <td class="time">14:15-15:15</td>
+        <td class="font-weight-bold">Lunch Break</td>
+
+      </tr>
+      <tr>
+        <td class="time">15:15-16:05</td>
+        <td><span class="text-success font-weight-bold">Simple is not easy - the overview od two
+            projects</span><br>MARIA KNOPS,
+          PhD<br>IF Research Polska</td>
+
+      </tr>
+      <tr>
+        <td class="time">16:10-17:00</td>
+        <td><span class="text-success font-weight-bold">TBA</span><br>HANNA NOWICKA, PhD Candidate<br>Oxford University
+        </td>
+      </tr>
+      <tr>
+        <td class="time">17:00-17:30</td>
+        <td class="font-weight-bold">Coffee Break</td>
+      </tr>
+      <tr>
+        <td class="time">17:30-18:20</td>
+        <td><span class="text-success font-weight-bold">TBA</span><br>ESTERA KOT, PhD Candidate<br>City & Warsaw
+          University of Technology</td>
+      </tr>
+      <tr>
+        <td class="time">18:20-18:30</td>
+        <td class="font-weight-bold">Closing Remarks</td>
+      </tr>
+    </table>
+  </section>
+  <!-- End Agenda Section -->
+  <!-- Start Abstracts Section -->
+  <section class="container bg-light">
```

Comment:
This looks bad `<section class="container bg-light">`
we want section without background but container
or ` <section class="bg-light">
    <div class="container">`